rotkehlchen/chain/optimism_superchain/etherscan.py

```diff
@@ -52,10 +52,13 @@ def _additional_transaction_processing(
             # TODO: write this tx_receipt to DB so it doesn't need to be queried again
             # https://github.com/rotki/rotki/pull/6359#discussion_r1252850465
             tx_receipt = self.get_transaction_receipt(tx.tx_hash)
+            l1_fee = 0
             if tx_receipt is not None:
-                l1_fee = int(tx_receipt['l1Fee'], 16)
+                if 'l1Fee' in tx_receipt:
```

Comment:
can't those two if's be combined? Weird lint did not complain

Reply:
it is due to the logging statement